app/components/Resource/Services.js

```diff
@@ -65,8 +67,9 @@ class Service extends Component {
       { infoHidden ? null :
         <div className="service-application-process-container">
           <ul className="service--details">
+            <ServiceContactDetails email={service.email}/>
             <ServiceEligibility subject='How to apply' result={this.props.service.application_process}/>
-            <ServiceEligibilities subject='Eligibilities' eligibilities={this.props.service.eligibilities}/>
+            <ServiceEligibility subject='Eligibilities' result={this.props.service.eligibility}/>
```

Comment:
If `ServiceEligibilities` isn't being used anymore, then perhaps we should delete class?

Reply:
It's odd, we have an eligibilities table (tag style field linking to a service), and a column in services for eligibility (which is just a text field, and a tag style field)

We should really deprecate the text field and rely entirely on tags I'd think, but both fields have data on prod.

I'd keep the component for now, since the ServiceEligibilities component renders the list style of tags

Reply:
Got it, I agree with keeping the component, then. When we made the large DB schema changes earlier this year, we didn't fully migrate over to the new fields, so it'd be a good idea to make that a longer-term objective of the askdarcel-api project. @jjfreund